### DISCUSSION TITLE
WIN7系统完美运行蓝灯最新版本的方法

### DISCUSSION THREAD

**Discussion by ycjdixon** [getlantern/lantern-forum-cn#537] [General]
兄弟们：我终于找到WIN7系统运行新版蓝灯的方法了，经本人测试多日证实完美有效。最新正式版7.8.4和测试版7.8.5都可以安装使用。8.0.0测试版可以安装但不能使用，应该是软件本身问题，我在WIN11系统电脑上测试了8.0.0版本也是同样情况。
现在，我把方法分享给大家。
1.下载并安装VxKex。选择Release版本下载，安装，软件大小为2.85M。下载安装前请先退出360安全卫士，否则不能安装。其他防护软件我电脑上没有，所以不清楚是不是需要关闭。
2.找到蓝灯安装目录下的的可执行文件（后缀为exe)的，鼠标右键，选择“属性”->"VxKex"，勾选Enable VxKex for this program，也可以勾选Report a different version of Windows。
[图片](https://wx1.sinaimg.cn/large/7510f694gy1hoxgheekgwj20bo0dvwg3.jpg)
VxKex官网：https://github.com/vxiiduu/VxKex   
下载地址：https://github.com/vxiiduu/VxKex/releases  下载第二个KexSetup_Release_1_1_1_1375.exe

视频教程（英语）：https://www.youtube.com/watch?v=zl7AsxtoPV8&t=58s
@Mike777Khoo

**Comment by gl1031** [comment]
我就搞不懂烂灯还能不能玩！技术员做个W7版本完全就是分分钟的事儿！
一天天扯什么底层代码逻辑做不了！摆明了就是摆烂不想管！
整的付了钱的会员们还要自行去研究解决！
请问那些接近10年的老会员，哪个不是从WIN7系统过来的！
一声不吭不打招呼就给阉了！人家付费前你跟人家讲清楚了么？

**Reply by ycjdixon** [reply]
确实是这样，多么简单的事啊！技术员就是不管。无奈自己想办法。WIN7系统装上VxKex，可以运行一些只支持WIN10以上的软件。刚好我测试了蓝灯能运行。有了VxKex的支持很多人都想退回WIN7系统了。在我们看来WIN7最好用，WIN11比WIN10还要烂。

**Comment by MaxMKMax** [comment]
@ycjdixon 谢谢您的分享. 非常感激!
